extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -58,6 +58,7 @@ const ALLOWLIST_EVENT_IN_SANDBOX = [
   AnalyticsEventType.RENDER_START,
 ];
 
+export const PERSIST_EVENT_TIMESTAMP = 'persistSessionEvent';
```

Comment:
Are other config keys defined elsewhere? Seems like a stray.

Reply:
This is part of the optimization of not initializing the `SessionManger` if the publisher has not opted in at the trigger level. It is a stray, and we could do this work in `amp-analytics.js` to determine if the SessionManager is needed at initialization, but I didn't want to conflate the two files.

Reply:
I meant stray as in—are other config keys define elsewhere?

Reply:
No, should I change it back to string literal to match everywhere else?